### ISSUE TITLE
Document Clustering between KeyBERT and Sentence Transformer?

### ISSUE THREAD

**Issue by km5ar** [MaartenGr/KeyBERT#165]
I'm wondering if anyone compared the differences using KeyBERT vs Sentence Transformers for document clustering?

**Comment by MaartenGr** [comment]
KeyBERT itself is already using SentenceTransformers for extracting the document and word embeddings. It might be interesting to compare how well the clustering would be on the keyword embeddings compared to the document embedding but unfortunately I have not tried it out yet.

**Comment by km5ar** [comment]
@MaartenGr Yeah! I read your official doc a few month ago, I remember there were a section which you suggesting about first use KeyBERT then clustering, but recently I tried to find that section again, but not able to locate it anymore.

**Comment by MaartenGr** [comment]
I actually do not remember using writing in the documentation as such a use case with respect to KeyBERT. It may have been PolyFuzz but KeyBERT is not generally used for clustering unless word embeddings are clustered again.
